css/properties/overflow-clip-box.json

```diff
@@ -0,0 +1,210 @@
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "59"
+              },
+              "firefox_android": {
+                "version_added": "59"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": false,
+              "deprecated": false
+            }
+          }
+        }
+      },
+      "overflow-clip-box-block": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/CSS/overflow-clip-box-block",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "59"
+            },
+            "firefox_android": {
+              "version_added": "59"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": false
+          }
+        }
+      },
+      "overflow-clip-box-inline": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/CSS/overflow-clip-box-inline",
```

Comment:
This one does actually work, I thought pages in the `Mozilla` directory were supposed to be for non-web-facing features though? 🤔

Reply:
Both this one and the above link, I think they're under the `Mozilla` directory because they're non-standard CSS properties that have only ever been implemented in Firefox, and also does not have a `-moz-` prefix because it's been [proposed to the W3C CSSWG](https://developer.mozilla.org/en-US/docs/Mozilla/CSS/overflow-clip-box#Specifications).  It's weird.  XD